recipes/iwyu/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "include-what-you-use" %}
+{% set version = "0.17" %}
+{% set llvm_version = "13.0.1" %}
```

Comment:
From the website it looks like the major version should be sufficient. Any reason to pin stronger here?

Reply:
not sure if I am following you here, I followed this table here: https://github.com/include-what-you-use/include-what-you-use#how-to-build
could you explain a bit more about that pls?

Reply:
@xhochy just a friendly reminder about this :)